docs/tutorials/substreams/getting-started.experimental.md

```diff
@@ -0,0 +1,318 @@
+-   [Helm for Linux](https://helm.sh/docs/intro/install/#from-script).
+-   [Helm for Mac](https://helm.sh/docs/intro/install/#from-homebrew-macos).
+
+Make sure `helm` is installed properly:
+
+```sh
+helm version
+```
+
+### Installing `minikube`
+
+Make sure you already install `docker`, if not you can follow our [Docker Quick-start](../docker/getting-started.md).
+
+-   [Minikube for Windows/Linux/Mac](https://minikube.sigs.k8s.io/docs/start/).
+
+Make sure `minikube` can start properly:
+
+```sh
+minikube version
+```
+```sh
+minikube version: v1.32.0
+commit: 8220a6eb95f0a4d75f7f2d7b14cef975f050512d
+```
+
+### Installing `kubectl`
+
+-   [Kubectl for Windows](https://kubernetes.io/docs/tasks/tools/install-kubectl-windows).
+-   [Kubectl for Linux](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux).
+-   [Kubectl for Mac](https://kubernetes.io/docs/tasks/tools/install-kubectl-macos).
+
+Alternatively, you can use `minikube` to download the appropriate version of `kubectl`. You can start the download process with this command.
+
+```sh
+minikube kubectl -- version
+```
+
+## Starting / Stopping `minikube`
+
+For starting, make sure you have docker running. The result might be different the first time you run it.
+
+```sh
+minikube start
+```
+```sh
+😄  minikube v1.32.0 on Ubuntu 22.04 (amd64)
+✨  Using the docker driver based on existing profile
+👍  Starting control plane node minikube in cluster minikube
+🚜  Pulling base image ...
+🔄  Restarting existing docker container for "minikube" ...
+🐳  Preparing Kubernetes v1.28.3 on Docker 24.0.7 ...
+🔗  Configuring bridge CNI (Container Networking Interface) ...
+🔎  Verifying Kubernetes components...
+    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
+🌟  Enabled addons: storage-provisioner, default-storageclass
+🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
+```
+
+To test if the local cluster is working properly.
+
+```sh
+kubectl get po -A
+```
+```sh
+NAMESPACE     NAME                               READY   STATUS    RESTARTS      AGE
+kube-system   coredns-5dd5756b68-jrqxj           1/1     Running   1 (21m ago)   23m
+kube-system   etcd-minikube                      1/1     Running   2 (21m ago)   24m
+kube-system   kube-apiserver-minikube            1/1     Running   2 (21m ago)   24m
+kube-system   kube-controller-manager-minikube   1/1     Running   2 (21m ago)   24m
+kube-system   kube-proxy-hsbj4                   1/1     Running   1 (21m ago)   24m
+kube-system   kube-scheduler-minikube            1/1     Running   2 (21m ago)   24m
+kube-system   storage-provisioner                1/1     Running   3 (19s ago)   24m
+```
+
+For stopping.
+
+```sh
+minikube stop
+```
+```sh
+✋  Stopping node "minikube"  ...
+🛑  Powering off "minikube" via SSH ...
+🛑  1 node stopped.
+```
+
+Or if you want to clear any persistent data please use
+```sh
+minikube delete
+```
+```sh
+minikube stop
+```
+## Deploy `substreams-charts`
```

Comment:
Should write the explanation about what is `substreams-charts`.

Reply:
I only give 1 line explanation, since to explain what a helm chart or what is a deployment template is is too in-depth.
User should do their own research.